contribute/Local-K8s-Development-Guide.md

```diff
@@ -0,0 +1,414 @@
+# K8s Development Guide
+
+Development Guide for Mojaloop on a local Kubernetes installation via Minikube.
+
+Refer to the following git repo for more information on running Mojaloop on Kubernetes: https://github.com/mojaloop/helm
+## Local Minikube Deployment
+### Setup local K8s environment
```

Comment:
This might be a bit confusing to users where we say Local Minikube Deployment in one place and saying local K8s environment in other place as they both are the same in that we are using minikube for local kubernetes. Also will be better if we spell kubernetes instead of k8s as novice users to kubernetes may not understand k8s..

Reply:
Noted